content/projects/gsoc/2018/remoting-over-message-bus.adoc

```diff
@@ -41,6 +41,20 @@ link:https://docs.google.com/document/d/17vmPvDtMbHT7nTKRlGReFRgOtwVImhgsETLOGPW
 * Every Tuesday, 02:00-03:00 PM UTC
 * Every Friday, 02:00-03:00 PM UTC
 
+=== Documentation
+
+All the notes, meeting minutes, design documentation can be found at https://drive.google.com/open?id=1-sua3rHvz84I8OZKL_3VpTlgEAMoBEHO
+
 === Timeline
 
-(to be updated)
\ No newline at end of file
+First phase:
+
+* 14/05 - 18/05: Setting up project skeleton, implementation of publishers, subscribers.
+* 21/05 - 25/05: PoC - Implementation of command transport for the message bus which involves command invocation, RMI, classloading and data streaming.
+* 04/06 - 08/06: PoC - Implementation of command transport for the message bus which involves command invocation, RMI, classloading and data streaming.
+* 11/06 - 15/06: Phase 1 Evaluation
+
+** Do demonstration of the PoC.
```

Comment:
Maybe makes sense to take one from https://issues.jenkins-ci.org/browse/JENKINS-51338 (1 extra item + no "Goal" references)